README.md

```diff
@@ -24,18 +24,29 @@ Ideas for future features include:
 
 ## How to run locally
 
+Install Maven if you dont have it pre-installed . [See here](https://www.mkyong.com/maven/how-to-install-maven-in-windows/)
```

Comment:
could you link to https://maven.apache.org instead of an ad supported private individual's site here?

Reply:
sure